en/flight_controller/pixhawk4_mini.md

```diff
@@ -0,0 +1,113 @@
+# Pixhawk 4 Mini
+
+The *Pixhawk<sup>&reg;</sup> 4 Mini* autopilot is designed for engineers and hobbyists who are looking to tap into the power of *Pixhawk 4* but are working with smaller drones. *Pixhawk 4 Mini* takes the FMU processor and memory resources from the *Pixhawk 4* while eliminating normally unused interfaces. This allows the *Pixhawk 4 Mini* to be small enough to fit in a 250mm racer drone.
+
+*Pixhawk 4 Mini* was designed and developed in collaboration with Holybro<sup>&reg;</sup> and Auterion<sup>&reg;</sup>. It is based on the [Pixhawk](https://pixhawk.org/) **FMUv5** design standard and is optimized to run PX4 flight control software.
+
+<img src="../../assets/flight_controller/pixhawk4mini/pixhawk4mini_iso_1.png" width="350px" title="Pixhawk 4 Mini Iso" />
+
+## Quick Summary
+
+* Main FMU Processor: STM32F765
+  * 32 Bit Arm® Cortex®-M7, 216MHz, 2MB memory, 512KB RAM
+* On-board sensors:
+  * Accel/Gyro: ICM-20689
+  * Accel/Gyro: BMI055
+  * Magnetometer: IST8310
+  * Barometer: MS5611
+* GPS: ublox Neo-M8N GPS/GLONASS receiver; integrated magnetometer IST8310
+* Interfaces:
+  * 8 PWM outputs
+  * 4 dedicated PWM/Capture inputs on FMU
+  * Dedicated R/C input for CPPM
+  * Dedicated R/C input for Spektrum / DSM and S.Bus with analog / PWM RSSI input
+  * 3 general purpose serial ports
+  * 2 I2C ports
+  * 3 SPI buses
+  * 1 CANBuses for CAN ESC
+  * Analog inputs for voltage / current of battery
+  * 2 additional analog input
+* Power System:
+  * USB Power Input: 4.75~5.25V
+  * Servo Rail Input: 0~36V
+  * Max current sensing: 120A
+  * Max input voltage: 6V
+* Weight and Dimensions:
+  * Dimensions: 38x55x15.5mm
+* Other Characteristics:
+  * Operating temperature: -40 ~ 85°c
+
+Additional information can be found in the [*Pixhawk 4 Mini* Technical Data Sheet](https://github.com/PX4/px4_user_guide/raw/master/assets/flight_controller/pixhawk4mini/pixhawk4mini_technical_data_sheet.pdf).
+
+## Purchase
+Available soon.
+<!-- Order from [Holybro](https://shop.holybro.com/). -->
+
+## Interfaces
+
+![Pixhawk 4 Mini interfaces](../../assets/flight_controller/pixhawk4mini/pixhawk4mini_interfaces.png)
+
+> **Warning** The **RC IN** and **PPM** ports are for RC receivers only. These are powered! NEVER connect any servos, power supplies or batteries (or to any connected receiver).
+
+
+## Pinouts
+
+Download *Pixhawk 4 Mini* pinouts from [here](https://github.com/PX4/px4_user_guide/raw/master/assets/flight_controller/pixhawk4mini/pixhawk4mini_pinouts.pdf).
+
+## Dimensions
+
+![Pixhawk 4 Dimensions](../../assets/flight_controller/pixhawk4mini/pixhawk4mini_dimensions.png)
+
+## Voltage Ratings
+
+*Pixhawk 4 Mini* can be redundant on the power supply if two power sources are supplied. The power rails are: **POWER** and **USB**.
+
+> **Note** The output power rail **MAIN OUT** does not power the flight controller board (and is not powered by it). You must supply power to one of **POWER** or **USB** or the board will be unpowered.
```

Comment:
Perhaps link "supply power" in the second sentence to the Power section in the quick start?

Reply:
Good idea.